File: app/src/main/java/com/emika/app/data/network/pojo/company/Payload.java
package com.emika.app.data.network.pojo.company;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Payload {

    @SerializedName("invited")
    @Expose
    private Boolean invited;
    @SerializedName("invitations")
    @Expose
    private List<Invitation> invitations = null;

    public Boolean getInvited() {
        return invited;
    }

    public void setInvited(Boolean invited) {
        this.invited = invited;
    }

    public List<Invitation> getInvitations() {
        return invitations;
    }

    public void setInvitations(List<Invitation> invitations) {
        this.invitations = invitations;
    }
}

File: app/src/main/java/com/emika/app/data/network/pojo/comment/Payload.kt
package com.emika.app.data.network.pojo.comment


import com.google.gson.annotations.SerializedName

data class Payload(
    @SerializedName("company_id")
    val companyId: String,
    @SerializedName("created_at")
    val createdAt: String,
    @SerializedName("created_by")
    val createdBy: String,
    @SerializedName("_id")
    val id: String,
    @SerializedName("task_id")
    val taskId: String,
    @SerializedName("text")
    val text: String,
    @SerializedName("updated_at")
    val updatedAt: String
)